catalyst/contrib/datasets/cifar.py

```diff
@@ -8,6 +8,7 @@
 import torch.utils.data as data
 
 from catalyst.contrib.datasets.functional import _check_integrity, download_and_extract_archive
+from catalyst.data.dataset import metric_learning
```

Comment:
```suggestion
from catalyst.data.dataset.metric_learning import MetricLearningTrainDataset
```
let's use full paths in the framework